### PR TITLE
Fix macro usage in `test_target_update_to`

### PR DESCRIPTION
**Pull request by jhdavis8**
For issue #10 , update macro usage, header comment, and whitespace mistakes.

Needs testing still. Summit is busy.

### CODE REVIEW

File: tests/4.5/target_update/test_target_update_to.c [hunk 1]
```diff
@@ -1,78 +1,80 @@
...
-        for (j = 0; j < N; j++) {
-          c[j] = (a[j] + b[j]);//c=12 
-        }
-  } // end target
-
-  update_b();
-  #pragma omp target update to(b[:N]) //update b=4 
+#pragma omp target
+    {
+      isHost = omp_is_initial_device();
```
Comment (josemonsalve2) [inline tests/4.5/target_update/test_target_update_to.c]: isHost is not needed anymore

File: tests/4.5/target_update/test_target_update_to.c [hunk 2]
```diff
@@ -1,78 +1,80 @@
...
-  }
-
   // a and b array initialization
   for (i = 0; i < N; i++) {
-    b[i] = 2; 
+    b[i] = 2;
     c[i] = 0;
   }
+
 #pragma omp target data map(to: a[:N], b[:N]) map(from: c) map(tofrom: isHost)
```
Comment (josemonsalve2) [inline tests/4.5/target_update/test_target_update_to.c]: is Host is not needed anymote

File: tests/4.5/target_update/test_target_update_to.c [hunk 3]
```diff
@@ -1,78 +1,80 @@
...
+  }
 }
-  
+
 // Test for OpenMP 4.5 target update with to
 int main() {
-  int errors= 0, i = 0, isHost = -1, isOffloading = 0, change_flag=0;
+  int errors= 0;
+  int i = 0;
+  int isHost = -1;
```
Comment (josemonsalve2) [inline tests/4.5/target_update/test_target_update_to.c]: isHost is not needed anymore

File: tests/4.5/target_update/test_target_update_to.c [hunk 4]
```diff
@@ -1,78 +1,80 @@
...
+  for (i = 0; i < N; i++) {
+    OMPVV_TEST_AND_SET_VERBOSE(errors, c[i] != 16);
+  }
 
-  if (!errors)
-    printf("Test passed on %s.\n", (isOffloading ? "device" : "host"));
-  else
-    printf("Test failed on %s\n", (isOffloading ? "device" : "host"));
+  OMPVV_INFOMSG_IF(!errors, "Test passed on %s.\n", (isOffloading ? "device" : "host"));
+  OMPVV_ERROR_IF(errors, "Test failed on %s\n", (isOffloading ? "device" : "host"));
```
Comment (josemonsalve2) [inline tests/4.5/target_update/test_target_update_to.c]: The host/device disctintion is done by the report, based on the OMPVV_TEST_OFFLOADING result. 

These messages are not needed.

**Comment by jhdavis8** [comment]
I removed the unneeded messages and variables, and tested on summit. It is passing all compilers

File: tests/4.5/target_update/test_target_update_to.c [hunk 5]
```diff
@@ -1,78 +1,74 @@
...
-  {
-    isOffloading = !omp_is_initial_device();
-  }
-
   // a and b array initialization
   for (i = 0; i < N; i++) {
-    b[i] = 2; 
+    b[i] = 2;
     c[i] = 0;
   }
```
Comment (josemonsalve2) [inline tests/4.5/target_update/test_target_update_to.c]: These two can be merged?

Comment (jhdavis8) [reply tests/4.5/target_update/test_target_update_to.c]: Ah yes I'll go ahead and do that.

**Comment by josemonsalve2** [comment]
Running and passing on GCC and Clang on Fatnode

**Pull request merged**